pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -64,6 +68,63 @@ const (
 	remotePowerShellCmdPrefix = "powershell.exe -NonInteractive -ExecutionPolicy Bypass "
 )
 
+// FileInfo contains information about a file
+type FileInfo struct {
+	Path   string
+	SHA256 string
+}
+
+// NewFileInfo returns a pointer to a FileInfo object created from the specified file
+func NewFileInfo(path string) (*FileInfo, error) {
+	contents, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, errors.Wrapf(err, "could not get contents of file")
+	}
+	return &FileInfo{
+		Path:   path,
+		SHA256: fmt.Sprintf("%x", sha256.Sum256(contents)),
+	}, nil
+}
+
+// filesToTransfer is a map of what files should be copied to the Windows VM and where they should be copied to
+var filesToTransfer map[*FileInfo]string
+
+func init() {
+	if err := populateFilesToTransfer(); err != nil {
+		log.Error(err, "could not populate map of files to transfer, will try again later")
+	}
+}
+
+// populateFilesToTransfer populates the filesToTransfer map with the proper values
+func populateFilesToTransfer() error {
+	if filesToTransfer != nil {
+		return nil
+	}
+	filesToTransfer = make(map[*FileInfo]string)
+	srcDestPairs := map[string]string{
+		wkl.IgnoreWgetPowerShellPath: remoteDir,
+		wkl.WmcbPath:                 k8sDir,
+		wkl.HybridOverlayPath:        k8sDir,
+		wkl.HNSPSModule:              remoteDir,
+		wkl.FlannelCNIPluginPath:     cniDir,
+		wkl.WinBridgeCNIPlugin:       cniDir,
+		wkl.WindowsExporterPath:      k8sDir,
```

Comment:
I need to change the windows exporter be a special case that is not required

Reply:
After looking again, `WindowsExporterPath` is listed as a required file and so we should require it to be transferred properly